ldk/javascript/src/webpack/generate-banner.ts

```diff
@@ -1,16 +1,53 @@
-export function generateMetadata(): string {
+import { LoopPermissions } from './loop-permissions';
+
+function isPermissionDefined(permission: string[]): boolean {
+  if (permission.length === 0 || 
+      permission === undefined) {
+    return true;
+  } 
+  return false;
+}
+
+function buildUrlPermissions(permissions: LoopPermissions): string {
```

Comment:
I'd actually like this to return `string[]`. If we return it as a single string then we're asking the loop metadata processor to split the strings.